packages/core/src/config/minimal.ts

```diff
@@ -18,6 +18,7 @@ export default {
     'operation-description': 'off',
     'operation-2xx-response': 'warn',
     'operation-4xx-response': 'off',
+    'enforcements': 'warn',
```

Comment:
should we turn off enforcements for `minimal` configuration ?

Reply:
good question, I think yes, we should do that. @RomanHotsiy @andriyl are you also agree?

Reply:
what are enforcements?

Reply:
Sorry, the PR description started with `enforcement` (but I think that's a typo in the PR description).

Reply:
@slavikbez @ohorbachevskyi I don't understand. What is it changing? If there are no enforcements defined in .redocly.yaml it's not going to report anything anyways. No?

@adamaltman what do you mean?